docs/Homebrew-brew-Maintainer-Guide.md

```diff
@@ -0,0 +1,105 @@
+# Homebrew/brew Maintainer Guide
+
+This document describes a few components of the `Homebrew/brew` repository that are useful for maintainers to
+be aware of, but don't necessarily need to appear in documentation for most users and contributors.
+
+## Reviewing PRs
+
+Using `gh pr checkout NUMBER` is a super easy way to check out a PR branch using the GitHub CLI.
+
+When reviewing a PR, use "comment", "approve", or "request changes" when submitting based on the following guidelines:
+
+- Comment: if the PR isn't quite ready to be merged
+- Approve: if you feel that the PR is in a good state to be merged, even if there are
+  non-blocking changes you'd like to be made
+- Request changes: if you feel strongly that the PR is likely to cause a problem for users or
+  have another reason to oppose the PR.
+
+## Merging PRs
+
+Merging is be done using the standard Merge button in the `Homebrew/brew` repository to preserve
```

Comment:
```suggestion
Merging is done using the standard Merge button in the `Homebrew/brew` repository to preserve
```

Reply:
Should it be this or

```suggestion
Merging is to be done using the standard Merge button in the `Homebrew/brew` repository to preserve
```

(not sure I have a preference)

Reply:
I thought of that too. However, I'd err on using fewer words when they're sufficient.